asistente_ladm_col/utils/quality_error_db_utils.py

```diff
@@ -70,14 +70,17 @@ def get_quality_error_connector(self, output_path, timestamp, load_layers=False)
         db = GPKGConnector(db_file)
         ili2db = Ili2DB()
         error_model = LADMColModelRegistry().model(LADMNames.QUALITY_ERROR_MODEL_KEY)
-        res, msg = ili2db.import_schema(db, [error_model.full_name()])
+
+        configuration = ili2db.get_import_schema_configuration(db, [error_model.full_name()])
+        res, msg = ili2db.import_schema(db, configuration)
 
         self.progress_changed.emit(50)
 
         if res:
             for catalog_key, catalog_xtf_path in error_model.get_catalogs().items():
                 logger.info(__name__, "Importing catalog '{}' to quality error database...".format(catalog_key))
-                res_xtf, msg_xtf = ili2db.import_data(db, catalog_xtf_path)
+                ili2db.get_import_data_configuration(db, catalog_xtf_path)
+                res_xtf, msg_xtf = ili2db.import_data(db, configuration, catalog_xtf_path)
```

Comment:
I guess the `catalog_xtf_path` parameter is not needed when calling `import_data`.